t/01_style.t

```diff
@@ -12,4 +12,6 @@ ok system(qq{git grep -I -l '[#/ ]*SPDX-License-Identifier ' ':!t/01_style.t'})
 $out = qx{git grep -ne "check_var('ARCH',.*)" -e "check_var('BACKEND',.*)" ':!lib/Utils/Architectures.pm' ':!lib/Utils/Backends.pm' 'lib' 'tests'};
 ok $? != 0 && $out eq '', 'No check_var function to verify ARCH/BACKEND types' or diag $out;
 ok system(qq{git grep -I -l \\( -e "egrep" -e "fgrep" \\) ':!t/01_style.t' ':!CONTRIBUTING.md'}) != 0, 'No usage of the deprecated egrep and fgrep commands';
+$out = qx{git grep -I -l 'nots3cr3t' ':!data/wsl/Autounattend_*.xml' 'data' | xargs grep -L 'luks[1,2]: { password: "nots3cr3t" }'};
```

Comment:
This might work to exclude it in general (I'm not aware of other encryption stuff at the moment) but this will only fly for jsonnet, you might need to cover too this, basically putting optional quotes in the regex:
`"luks2": { "password": "my secret passphrase" }`

Reply:
The first command will get files list which include 'nots3cr3t', The second command, it will do search from the first result, and check if those files include luks[1,2]: { password: "nots3cr3t" }.   So, it can search both json and jsonnet files.

Reply:
the regex includes `password` without double quotes, how can detect that, have you tried out to include a file like that?

Reply:
I changed to use 'luks.*password.*nots3cr3t'. It will match your requirement.